tests/test_settings.py

```diff
@@ -0,0 +1,48 @@
+import os
+import shutil
+
+from bin.bb8.settings import Settings
+from bin.bb8.targets import NamedVolumeTarget, TargetOptions, DirectoryTarget
+
+json = """{
+    "starport": {
+        "addr": "the moon",
+        "ip": "127.0.0.1",
+        "user": "astronaut",
+        "backup_location": "houston"
+    },
+    "targets": [
+        {
+            "name": "target_1",
+            "type": "named_volume",
+            "volume": "volume_name"
+        },
+        {
+            "name": "target_2",
+            "type": "directory",
+            "path": "/some/path",
+            "backup": false,
+            "restore": false
+        }
+    ]
+}"""
+
+
+def test_can_parse_settings():
+    with open('test.json', 'w') as f:
+        f.write(json)
+    s = Settings('test.json')
+    assert s.starport == {
+        "addr": "the moon",
+        "ip": "127.0.0.1",
+        "user": "astronaut",
+        "backup_location": "houston"
+    }
+    assert s.targets[0] == NamedVolumeTarget("target_1", "volume_name",
+                                             TargetOptions(True, True))
+    assert s.targets[1] == DirectoryTarget("target_2", "/some/path",
+                                           TargetOptions(False, False))
+
+
+def teardown_module(module):
```

Comment:
If this is created within a test, should it not be torn down in that test too (or set up at the module level?)

Reply:
Moved the setup to the module level. If we add more tests to this module we may have to rethink.

Reply:
Cool, this is symmetric which is nice